docker/openemr/flex-3.10/utilities/devtools.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/sh
+
```

Comment:
lets also add another tool called "build-themes" (and is not included with clean-sweep) that will do something like
```
if [ "$1" == "build-themes" ]; then
    echo "Building OpenEMR Themes"
    cd /var/www/localhost/htdocs/openemr
    npm run build
fi
```